subgraphs/aave-forks/schema.graphql

```diff
@@ -53,6 +53,12 @@ type Token @entity {
 
   " Optional field to track the block number of the last token price "
   lastPriceBlockNumber: BigInt
+
+  " the market ID for underlying/asset token, aToken, vToken and sToken"
+  _market: Market
```

Comment:
Since we are breaking the schema here, we might as well make these changes as well: https://github.com/messari/subgraphs/pull/1220

Reply:
OK, that makes sense.